.changeset/biome_deserialize_context

```diff
@@ -0,0 +1,54 @@
+---
+biome_deserialize: major
+biome_deserialize_macros: major
```

Comment:
Since we are still in `0.*`, we're allowed to create a minor and make a breaking change.